README.rst

```diff
@@ -0,0 +1,73 @@
+Django Preview Image Widget
+===========================
+
+    An ImageField preview widget application
+
+|status-image| |coverage-image|
+
+An application for displaying the image preview of a ImageField in a DjangoForm
```

Comment:
Yes, this uses bootstrap in button classes, this must to be added to the dependencies